### PR TITLE
Loading taskflow tasks

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

**Comment by cjh1** [comment]
@kotfic Can you take a look at this and try it out?

File: cumulus/taskflow/utility/__init__.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                yield module
+        else:
+            yield '%s%s' % (prefix,  name)
+
+
+def find_taskflow_modules():
+    # There has to be a better way!
+    base_path = os.path.dirname(pr.resource_filename(cumulus.__name__,
+                                                     '__init__.py'))
+    base_path = os.path.abspath(os.path.join(base_path, '..'))
```
Comment (kotfic) [inline cumulus/taskflow/utility/__init__.py]: I'm fairly sure this won't work with cumulus packaged as an egg or a wheel. We talked about moving this inside the ```cumulus``` namespace?

File: cumulus/conf/default_config.json [hunk 2]
```diff
@@ -11,6 +11,9 @@
      "moabReader": {
             "pluginPath": "/test"
     },
-    "amis": {}
+    "amis": {},
+    "taskflow": {
+        "path": ["tests/integration/taskflow"]
```
Comment (kotfic) [inline cumulus/conf/default_config.json]: copying the default into my config.json,  I end up getting task_runner as a task - is that expected behavior?

**Comment by cjh1** [comment]
@kotfic I have pushed the updates we talked about

**Comment by kotfic** [comment]
@cjh1  LGTM.  AttributeError problems with passing taskflow tasks to default celery queue are a separate issue.

**Pull request merged**